packages/reason-relay/src/ReasonRelay.re

```diff
@@ -1178,14 +1182,20 @@ exception Mutation_failed(array(mutationError));
 
 [@bs.module "relay-runtime"]
 external commitMutation_:
```

Comment:
same as for types, we could follow the same naming convention or just use shadowing to be sure that the raw binding is not used elsewhere.

Reply:
Agreed! And separate-pr-material 🤪

Reply:
haha yeah, a new cleanup PR for `ReasonRelay.re` in perspective :)